### PR TITLE
Changed "Thomaeum Kempen" to "isi_ko404"

### PR DESCRIPTION
**Pull request by isiko**
# Description
Changed the mention of "Thomaeum Kempen" as a contributor to "isi_ko404", as Thomaeum is just my school.

(see: 84cb16892ff37d1b2940f512c327e9dd5c4585c7)

### CODE REVIEW

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Can you please put your name in alphabetical order? In the `README.md` please reformat the names if needed to have a pleasant block when being displayed at GitHub.

File: README.md [hunk 1]
```diff
@@ -111,11 +111,12 @@ Copyright 2011-2015 Daniel Dorau
...
-Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista,
-Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase,
-Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal,
-Teeranai.P, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
+Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, isi_ko404, 
+Jasper van der Graaf,Joergi, koelnkalkverbot, Larissa Yasin, ligi, 
+Luis Azcuaga, Mateus Baptista, Matthias Geisler, Matthias Hunstock,
+ ichaelRocks, Nghiem Xuan Hien, NiciDieNase, Noemis, Omicron, Poschi,
+ rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal, Teeranai.P,
+ Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
```
Comment (johnjohndoe) [inline README.md:119]: Almost nice - please check the preview here: https://github.com/isiko/EventFahrplan/tree/correctingContributionsNote

Please also squash your changes into one commit and force push it onto this branch.

Comment (isiko) [reply README.md]: Sorry, I didn't See those whitespaces 😅

**Review by johnjohndoe — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -114,7 +114,7 @@
     <string name="copyright_notes" translatable="false">
         Copyright 2013-2022 johnjohndoe.
         \nCopyright 2011-2015 Daniel Dorau.
-        \nContributions from 0x5ubt13, Adriano Pereira Junior, Akarsh Seggemu, Andrea Marziali, Andy Scherzinger, Andreas Schildbach, Animesh Verma, bashtian, bjoernb, Björn Olsson Jarl, ButterflyOfFire, cacarrara, Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, Jasper van der Graaf, Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista, Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase, Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal, Teeranai.P, Thomaeum Kempen, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka.
```
Comment (johnjohndoe) [inline app/src/main/res/values/strings.xml:117]: Please also squash your changes into one commit and force push it onto this branch. You can simple `git reset HEAD~4` the commits on your branch and create a single new commit. Then `git push --force origin correctingContributionsNote` it to update this PR.

Comment (isiko) [reply app/src/main/res/values/strings.xml:117]: I'll do that ASAP, the only problem is that my laptop's battery is currently dead 🤦

**Comment by isiko** [comment]
Whoops, wrong command

File: README.md [hunk 3]
```diff
@@ -111,11 +111,12 @@ Copyright 2011-2015 Daniel Dorau
...
 bashtian, bjoernb, Björn Olsson Jarl, ButterflyOfFire, cacarrara,
-Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, Jasper van der Graaf,
-Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista,
-Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase,
-Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal,
-Teeranai.P, Thomaeum Kempen, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
+Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, isi_ko404, 
+Jasper van der Graaf,Joergi, koelnkalkverbot, Larissa Yasin, ligi, 
+Luis Azcuaga, Mateus Baptista, Matthias Geisler, Matthias Hunstock,
+ichaelRocks, Nghiem Xuan Hien, NiciDieNase, Noemis, Omicron, Poschi,
```
Comment (johnjohndoe) [inline README.md:117]: Please restore the `M` in `MichaelRocks`.

File: README.md [hunk 4]
```diff
@@ -111,11 +111,12 @@ Copyright 2011-2015 Daniel Dorau
 Contributions from 0x5ubt13, Adriano Pereira Junior, Akarsh Seggemu,
 Andrea Marziali, Andy Scherzinger, Andreas Schildbach, Animesh Verma,
 bashtian, bjoernb, Björn Olsson Jarl, ButterflyOfFire, cacarrara,
-Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, Jasper van der Graaf,
-Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista,
-Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase,
-Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal,
-Teeranai.P, Thomaeum Kempen, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
+Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, isi_ko404, 
+Jasper van der Graaf,Joergi, koelnkalkverbot, Larissa Yasin, ligi,
```
Comment (johnjohndoe) [inline README.md:115]: Please restore the whitespace between `Jasper van der Graaf,Joergi`.

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Please fix and `git commit --amend` these minor issues.

**Comment by johnjohndoe** [comment]
Thanks for the fix. You can avoid the multiple commits being pushed to the branch when you don't pull your remote branch but overwrite it with your local state via `git commit --force`. I am going to fix it for this branch - no action needed by you.

---

One note about resolving conversations: Please leave them open for the reporter (in this case me) to check and resolve them. You can post a comment or just a :heavy_check_mark: to indicate that you took care of it. Thank you. No offense.

**Review by johnjohndoe — APPROVED** [review]

**Comment by isiko** [comment]
Thank you for your patience 😅

**Pull request merged**